src/url.js

```diff
@@ -190,25 +194,6 @@ export function parseQueryString(queryString) {
 
 
 /**
- * Don't use this directly, only exported for testing. The value
- * is available via the origin property of the object returned by
- * parseUrl.
- * @param {string|!Location} url
- * @return {string}
- * @visibleForTesting
- */
-export function getOrigin(url) {
```

Comment:
That was my first try. Then this function become so slim so that I decided to inline it :-).